projectroles/tests/test_templatetags.py

```diff
@@ -453,15 +451,27 @@ def test_get_backend_plugins(self):
 
     # TODO: Test get_site_app_messages() (set up admin alert)
 
-    def test_has_star(self):
-        """Test has_star()"""
-        # Test with no star
+    def test_star(self):
```

Comment:
As said above, these tests can be removed together with the template tag itself.